test/integration/user_checks_out_from_cart_test.rb

```diff
@@ -16,6 +16,12 @@ class UserChecksOutFromCartTest < ActionDispatch::IntegrationTest
       click_button "Login"
     end
 
+    StripeMock.start
+    customer = Stripe::Customer.create({
```

Comment:
Useless assignment to variable - `customer`.<br>Redundant curly braces around a hash parameter.